src/site/twitch.tv/modules/chat/ChatList.vue

```diff
@@ -132,7 +133,9 @@ const onMessage = (msgData: Twitch.AnyMessage): boolean => {
 };
 
 function onChatMessage(msg: ChatMessage, msgData: Twitch.AnyMessage, shouldRender = true) {
+	let shouldRenderMessage = shouldRender;
```

Comment:
Why copy `shouldRender` to a new variable, rather than simply using it? Function arguments are not constant

Reply:
Good practice is to not rewrite parameters and just copy them

Reply:
I'd be curious to know why you believe that's good practice. Seems to me this only complicates code and only serves to allocate one extra bit of memory for no actual gain.

Reply:
Mutating might have some unintended side effects and immutability protects against this. It makes me uneasy when reassigning parameters :)

Reply:
It will have no side effects here. Due to this being one of the most called functions, currently I don't want a change like this. It actually provides no tangible benefit.